llpc/util/llpcElfWriter.cpp

```diff
@@ -735,6 +737,285 @@ template <class Elf> void ElfWriter<Elf>::updateMetaNote(Context *pContext, cons
   pNewNote->data = data;
 }
 
+// =====================================================================================================================
+// Get the count of relocs in the reloc section
+//
+// @param pRelocSection : Reloc section
+template <class Elf> size_t ElfWriter<Elf>::getRelocsCount(const SectionBuffer *pRelocSection) {
```

Comment:
nit: the llpc/llvm coding style does not have the `p` prefix for pointer variables
`pRelocSection` -> `relocSection`

Reply:
Same for other functions in this class

Reply:
Addressed.